packages/perseus/src/widgets/__tests__/explanation.test.ts

```diff
@@ -41,167 +85,184 @@ describe("Explanation", function () {
+        );
 
-        // Assert
-        // get asserts if it doesn't find a single matching element
-        expect(screen.getByText("This is an explanation")).toBeVisible();
+        // Assert - transition when revealing
+        expect(screen.getByTestId("content-container").className).toContain(
+            "transitionExpanded",
+        );
 
-        // Act - collapse with a click
-        const collapseButton = screen.getByRole("button", {
-            name: "[Hide explanation!]",
-        });
-        collapseButton.focus();
-        await userEvent.keyboard("{Enter}");
+        // Act - collapse
+        await userEvent.click(
+            screen.getByRole("button", {name: "Hide explanation!"}),
+        );
 
-        // Assert
-        expect(screen.queryByText("This is an explanation")).toBeNull();
+        // Assert - transition when concealing
+        expect(screen.getByTestId("content-container").className).toContain(
+            "transitionCollapsed",
+        );
+
+        // Clean-up
+        matchMediaMock.mockRestore();
     });
 
-    it("can be expanded and collapsed with the keyboard - Space bar", async function () {
+    it("does NOT use transitions when the user prefers reduced motion", async () => {
         // Arrange
+        const matchMediaMock = jest
+            .spyOn(window, "matchMedia")
+            .mockImplementation(
+                getMatchMediaMockFn(
+                    false,
+                    "(prefers-reduced-motion: no-preference)",
+                ),
+            );
         renderQuestion(question1);
 
-        // Act - expand with a click
-        const expandButton = screen.getByRole("button", {
-            name: "[Explanation]",
-        });
-        expandButton.focus();
-        await userEvent.keyboard(" ");
+        // Act - expand
+        await userEvent.click(
+            screen.getByRole("button", {name: "Explanation"}),
+        );
 
-        // Assert
-        // get asserts if it doesn't find a single matching element
-        expect(screen.getByText("This is an explanation")).toBeVisible();
+        // Assert - transition when revealing
+        expect(screen.getByTestId("content-container").className).not.toContain(
+            "transitionExpanded",
+        );
 
-        // Act - collapse with a click
-        const collapseButton = screen.getByRole("button", {
-            name: "[Hide explanation!]",
-        });
-        collapseButton.focus();
-        await userEvent.keyboard(" ");
+        // Act - collapse
+        await userEvent.click(
+            screen.getByRole("button", {name: "Hide explanation!"}),
+        );
 
-        // Assert
-        expect(screen.queryByText("This is an explanation")).toBeNull();
+        // Assert - transition when concealing
+        expect(screen.getByTestId("content-container").className).not.toContain(
+            "transitionCollapsed",
+        );
+
+        // Clean-up
+        matchMediaMock.mockRestore();
```

Comment:
Jest should be cleaning up all mocks at the end of each test, so this isn't needed.

```suggestion
```